### PR TITLE
Game statistics

### PR DESCRIPTION
**Pull request by LelandCurtis**
complete all game_stat methods and tests. Prep stat_tracker class and spec file for all methods.

### CODE REVIEW

File: data/teams_test.csv [hunk 1]
```diff
@@ -0,0 +1,15 @@
+team_id,franchiseId,teamName,abbreviation,Stadium,link
```
Comment (kevingloss) [inline data/teams_test.csv:1]: May run into issues testing if we reference a team that isn't included in this sample file. Just something to keep in mind.

**Review by echon006 — APPROVED** [review]
Looks good

File: spec/stat_tracker_spec.rb [hunk 2]
```diff
@@ -5,10 +5,24 @@
...
+  #   }
+  #
+  #   @stat_tracker = StatTracker.from_csv(locations)
+  # end
+
   before(:all) do
-    game_path = './data/games.csv'
-    team_path = './data/teams.csv'
-    game_teams_path = './data/game_teams.csv'
+    game_path = './data/games_test.csv'
```
Comment (dkulback) [inline spec/stat_tracker_spec.rb:23]: This will make the rspec super fast!

**Review by kevingloss — APPROVED** [review]
Looks good!

**Review by dkulback — APPROVED** [review]

**Pull request merged**